NightscoutServiceKit/NightscoutService.swift

```diff
@@ -331,25 +335,24 @@ extension NightscoutService: RemoteDataService {
         uploader.uploadProfiles(stored.compactMap { $0.profileSet }, completion: completion)
     }
     
```

Comment:
When a Push Notification arrives, we will try to identify which source it was generated from. Then use that source to extract a RemoteCommand.